### PR TITLE
Add OSCAL content and configure CI processing.

### LINKED ISSUE #3: Configure CI
We should frame out some initial CI linters.

### PR DESCRIPTION
**Pull request by xee5ch**
Closes #3.

### CODE REVIEW

**Comment by xee5ch** [comment]
There is much work to be done @JonZeolla, but perhaps I can come to the bi-weekly sync, talk about how I would envision catalog processing work in a CI system to meet your needs. I also have some questions about controls. Sounds good?

**Comment by JonZeolla** [comment]
@xee5ch sounds great!  Also happy to discuss anything async as you have time.  Whatever is easiest for you

**Comment by xee5ch** [comment]
> @xee5ch have you had a chance to look at this lately? Let me know if I can help - thanks!

Sorry it has been so long, I will get back on this tonight and during the course of the week!

**Comment by JonZeolla** [comment]
@xee5ch can you share a bit of how you're generating these outputs?  I would love something we could put as a github action or have others run to perform updates.  That will be especially useful for the CNSWPv3 work

**Comment by xee5ch** [comment]
> @xee5ch can you share a bit of how you're generating these outputs? I would love something we could put as a github action or have others run to perform updates. That will be especially useful for the CNSWPv3 work

Let me plug away at this and we meet later this week?

**Comment by xee5ch** [comment]
OK, @JonZeolla, let's meet again later this week to sync up. trestle looks viable with the PR as-is, but the output is messy for now.

https://gist.github.com/xee5ch/f74bbc8d4cafcff0e6413d32a3a102e0

I am not sure about what is the "right way" (or maybe only way) to clean up all the `null`s (it'd be super preferable to just drop any JSON object map property that is `null` and not even specify them) and we need to talk about the metadata info and backmatter stuff. The latter things are a matter of taste, and we can hard-code them into the Python tool or we can parameterize those in a settings file and work around it.

File: csv_to_oscal.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import csv
+from datetime import datetime, timezone
+from json import dumps
+import logging
+from os import PathLike
```
Comment (JonZeolla) [inline csv_to_oscal.py:7]: `pathlib`? /shrug

**Comment by JonZeolla** [comment]
Looks like we need to fill out - some examples below, but look at the [JSON schema](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-outline/) (good example [here](https://github.com/usnistgov/oscal-content/blob/main/nist.gov/SP800-53/rev5/yaml/NIST_SP-800-53_rev5_LOW-baseline_profile.yaml#L1-L35)):
- "published": Datetime
- "revisions": [Github URI]
- "document_ids": CNSWPv1
- "links": ["link to the white paper"]
- "roles": null,
- "locations": Insert website (github repo?)
- "parties": null,
- "responsible_parties": null,
- "remarks": null

**Pull request merged**